src/scripts/main.js

```diff
@@ -1,3 +1,34 @@
 'use strict';
 
-// write code here
+const tHeadTags = document.querySelectorAll('th');
+const tbody = document.querySelector('tbody');
```

Comment:
Ensure that `tbody` is not null to prevent errors if the table body is missing from the DOM.